app/components/course-page/step-list.hbs

```diff
@@ -36,7 +36,18 @@
   {{#each this.otherStepGroups as |stepGroup|}}
     {{#if stepGroup.title}}
       <div class="px-3 py-2">
-        <h3 class="text-xs uppercase font-semibold text-gray-400 border-b">{{stepGroup.title}}</h3>
+        <h3 class="flex justify-between items-center text-xs uppercase font-semibold text-gray-400 border-b pb-0.5">
+          {{stepGroup.title}}
+
+          <Toggle
+            @isDisabled={{get @activeRepository "isNew"}}
+            @isOn={{(not @configureExtensionsModalIsOpen)}}
```

Comment:
@libmartinito the toggle state here is supposed to indicate whether the extension is enabled, so this can just be `true`, no?